README.md

```diff
@@ -2,3 +2,21 @@
 [![License Apache 2.0](https://img.shields.io/badge/license-Apache%202.0-green.svg)](https://github.com/ustwo/android-boilerplate/blob/master/LICENSE.md)
 [![ustwo twitter](https://img.shields.io/badge/twitter-@ustwo-blue.svg)](http://twitter.com/ustwo)
 [![Platform Android](https://img.shields.io/badge/platform-Android-blue.svg)](https://www.android.com)
+
+## Description
+Android boilerplate project to use as a reference when starting new Android projects. It provides basic architecture, tools and guidelines that we use when developing Android apps.
+
+## Modules
+This project consists of the following modules:
+
+- app, Phone & tablet module
```

Comment:
Yeah it's correct. Format is `[module name], [description]` and is boilerplate to ease adding more